packages/react-devtools-scheduling-profiler/src/App.js

```diff
@@ -7,22 +7,32 @@
  * @flow
  */
 
-import type {ReactProfilerData} from './types';
+// Reach styles need to come before any component styles.
+// This makes overriding the styles simpler.
+import '@reach/menu-button/styles.css';
+import '@reach/tooltip/styles.css';
 
 import * as React from 'react';
-import {useState} from 'react';
 
-import ImportPage from './ImportPage';
-import CanvasPage from './CanvasPage';
+import {ModalDialogContextController} from './ModalDialog';
+import {SettingsContextController} from './Settings/SettingsContext';
```

Comment:
I'm not convinced the settings dialog is worth the hassle of the forked contexts and files. For now, I'm going to just rip it out :D